### PR TITLE
BlueChip Chain Registry First Push

### PR DESCRIPTION
**Pull request by Bluechip23**
We are a chain focusing on a decentralized subscription model to enhance content creation.

### CODE REVIEW

File: BlueChip/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "BlueChip",
```
Comment (JeremyParish69) [inline BlueChip/assetlist.json:3]: Please remove all capital letters from directory names and `chain_name`s (BlueChip -> bluechip and Images -> images)

File: BlueChip/chain.json [hunk 2]
```diff
@@ -0,0 +1,80 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "BlueChip",
+    "status": "live",
+    "website": "https://www.bluechip.link",
+    "network_type": "mainnet",
+    "pretty_name": "BlueChip",
```
Comment (JeremyParish69) [inline BlueChip/chain.json:7]: Although the chain_name property cannot have capital letters, the pretty_name is what apps will display and can keep the capital letters

File: BlueChip/chain.json [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    "chain_id": "bluechip_1",
+    "bech32_prefix": "bcp",
```
Comment (JeremyParish69) [inline BlueChip/chain.json:9]: Please register this prefix: https://github.com/satoshilabs/slips

File: BlueChip/assetlist.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+            "denom": "ubluechip",
+            "exponent": 0
+          },
+          {
+            "denom": "ebl",
+            "exponent": 6
+          }
+        ],
+        "base": "ubluechip",
+        "name": "blue chip",
```
Comment (JeremyParish69) [inline BlueChip/assetlist.json:18]: Not a requirement, but I suspect Capital letter might be intended here...

File: BlueChip/assetlist.json [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    "assets": [
+      {
+        "description": "The native staking token of BlueChip.",
+        "denom_units": [
+          {
+            "denom": "ubluechip",
+            "exponent": 0
+          },
+          {
+            "denom": "ebl",
```
Comment (JeremyParish69) [inline BlueChip/assetlist.json:13]: This is likely incorrect, left over from a template assetlist json file. I assume this is meant to be 'bcp'

File: BlueChip/assetlist.json [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+        "display": "bcp",
+        "symbol": "BCP",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/BlueChip/images/BlueChip.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/BlueChip/images/BlueChip.svg"
+        },
+        "coingecko_id": "",
+        "images": [
+          {
+            "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/BlueChip/images/BlueChip.png",
```
Comment (JeremyParish69) [inline BlueChip/assetlist.json:28]: please lowercase the directory name and file name in these urls (the uploaded file name is 'bluechip.png', not BlueChip.png)

File: BlueChip/assetlist.json [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+          {
+            "denom": "ebl",
+            "exponent": 6
+          }
+        ],
+        "base": "ubluechip",
+        "name": "blue chip",
+        "display": "bcp",
+        "symbol": "BCP",
+        "logo_URIs": {
```
Comment (JeremyParish69) [inline BlueChip/assetlist.json:21]: Please remove logo_URIs, as this is a legacy property we are no longer adding to

File: BlueChip/assetlist.json [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/BlueChip/images/BlueChip.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/BlueChip/images/BlueChip.svg"
+        },
+        "coingecko_id": "",
```
Comment (JeremyParish69) [inline BlueChip/assetlist.json:25]: Please remove properties with empty values like this coingecko ID here.

File: BlueChip/chain.json [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+    "description": "Stake to the BlueChip chain, interact with the creator pools, and subscribe through the creator subscription contracts.",
+    "codebase": {
+      "git_repo": "https://github.com/Bluechip23/bluechip/tree/main",
+      "recommended_version": "v1",
+      "compatible_versions": [
+        "v1"
+      ],
+      "cosmos_sdk_version": "0.46.7",
+      "cosmwasm_enabled": true,
+
```
Comment (JeremyParish69) [inline BlueChip/chain.json:43]: Please remove this extra line

File: BlueChip/chain.json [hunk 10]
```diff
@@ -0,0 +1,80 @@
...
+          "recommended_version": "v1",
+          "compatible_versions": [
+            "v1"
+          ],
+          "cosmos_sdk_version": "0.46.7",
+          "cosmwasm_enabled": true,
+        }
+      ]
+    },
+    "logo_URIs": {
```
Comment (JeremyParish69) [inline BlueChip/chain.json:59]: Please remove this logo_URIs, too (but keep `images`)

File: BlueChip/chain.json (hunk 3 above)
Comment (Bluechip23) [reply BlueChip/chain.json:9]: Do I simply just add BlueChip's information to the chart?

Comment (JeremyParish69) [reply BlueChip/chain.json:9]: yes

**Comment by JeremyParish69** [comment]
Looks like you've made a new PR. Can his be closed then?

**Comment by Bluechip23** [comment]
Yes it can be.


Nothing but the best

Nick Davis,

[cid:886ede93-8ed9-4790-a8dd-5a753858c081]

________________________________

**Comment by Bluechip23** [comment]
Is there anything else you will need from me?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Pull request closed without merging**